client/tests/setup/mockHCaptcha.ts

```diff
@@ -0,0 +1,26 @@
+import { test as base, type BrowserContext, type Route } from '@playwright/test';
+
+export const test = base.extend<{ context: BrowserContext }>({
+  context: async ({ context }, use) => {
+    // Only mock hCaptcha in CI environment
+    if (process.env.CI) {
+      await context.route('**/hcaptcha.com/**', (route: Route) => route.fulfill({
+        status: 200,
+        body: `
+          <html>
+            <body>
+              <div>
+                <label>hCaptcha checkbox with text</label>
+                <input type="checkbox" checked="true" />
+              </div>
+            </body>
+          </html>
+        `,
+        contentType: 'text/html'
+      }));
+    }
+    await use(context);
+  },
+});
+
+export { expect } from '@playwright/test'; 
```

Comment:
what is the purpose of exporting the `expect` here vs how it was initially being imported?

Reply:
This was just to get test and expect encapsulated and gotten from one place. But tbh this mock captcha might not fix the issues I'm running into.